src/analysis/utils/cost-utils.ts

```diff
@@ -48,41 +47,51 @@ import { cloudProviderTypes } from 'src/libs/workspace-utils';
 // GOOGLE COST METHODS begin
 
 export const dataprocCost = (machineType: string, numInstances: number): number => {
+  // console.log('dataprocCost', { machineType, numInstances });
```

Comment:
uncommenting these will let us follow the cost logic - testing of this area is sparse and does not reliably catch defects